foundationdb/src/transaction.rs

```diff
@@ -613,6 +613,8 @@ impl Drop for TransactionInner {
         }
     }
 }
+unsafe impl Send for TransactionInner {}
```

Comment:
I can believe that Database is threadsafe, but is transaction as well?

Reply:
There's an implication in a documentation.

https://docs.rs/foundationdb/0.1.0/foundationdb/options/enum.TransactionOption.html

NextWriteNoWriteConflictRange
Care needs to be taken when using this option on a transaction that is shared between multiple threads. When setting this option, write conflict ranges will be disabled on the next write operation, regardless of what thread it is on.